### PR TITLE
Added python-simplegeneric.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

File: recipes/python-simplegeneric/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - simplegeneric
+
+about:
+  home: http://cheeseshop.python.org/pypi/simplegeneric
+  license: Zope Public License
+  summary: "Simple generic functions (similar to Python's own len(), pickle.dump(), etc.)"
+
+extra:
+  recipe-maintainers:
+    - minrk
```
Comment (pelson) [inline recipes/python-simplegeneric/meta.yaml]: Given previous comments, I hope I'm not taking liberties adding you @minrk?

**Comment by msarahan** [comment]
You read my mind on needing this.  Any interest in adding more than an import test?

**Comment by pelson** [comment]
> Any interest in adding more than an import test?

Personally, not especially. If it imports, and it is pure python, what can go wrong? (I fully accept the irony of those words 😄 )

**Comment by pelson** [comment]
Want in @msarahan?

**Comment by msarahan** [comment]
Sure, happy to help where I can.  I think we're generally saying import-test only is OK for python-only packages, right?

**Comment by pelson** [comment]
>  I think we're generally saying import-test only is OK for python-only packages, right?

practicality beats purity 😉 

If there are useful (confidence inspiring) tests to be added, I'd add them, but in this case, I believe there would be nothing gained from adding the full test suite.

Thanks for the feedback @msarahan.

**Comment by pelson** [comment]
I forgot a super important detail here - I'm sorry for not raising it before merging.

simplegeneric was ambiguous so I went for python-simplegeneric:

 * https://pypi.python.org/pypi/simplegeneric
 * https://www.npmjs.com/package/simplegeneric

This is a break from the existing name in anaconda. Ping @ocefpaf and @jakirkham. I'm happy to be led back onto the anaconda path if the weight of consensus wills it.

**Comment by msarahan** [comment]
I agree with your path here.

**Comment by ocefpaf** [comment]
> I agree with your path here.

Me too.

(I am always :+1: to using `python-` prefix for disambiguation.)

**Comment by jakirkham** [comment]
Also, agree with prefixing `python-`.

**Comment by pelson** [comment]
OK, thanks for the support. I'll track down any feedstocks which use the `simplegeneric` name, and update.

**Comment by jakirkham** [comment]
> If there are useful (confidence inspiring) tests to be added, I'd add them, but in this case, I believe there would be nothing gained from adding the full test suite.

Agreed. I'm way more stressed about rigorously testing compiled code than pure Python packages.

**Comment by ocefpaf** [comment]
> OK, thanks for the support. I'll track down any feedstocks which use the `simplegeneric` name, and update.

I believe it is only [`ipython`](https://github.com/conda-forge/ipython-feedstock/blob/master/recipe/meta.yaml#L27) that uses `simplegeneric`.

File: recipes/python-simplegeneric/meta.yaml (hunk 1 above)
Comment (minrk) [inline recipes/python-simplegeneric/meta.yaml]: 👍

**Comment by msarahan** [comment]
Sorry for my consent on the prefix here.  I think in general, we should always match defaults, wherever defaults already has a package. =(

I hope the whole conda namespaces thing will sort this out soon, but I think it is a necessary precaution for now.

**Comment by ocefpaf** [comment]
> Sorry for my consent on the prefix here. 

It made sense at the time. Let's talk about this again in our next meeting as the `namespaces` changed the rules of the game here.

**Pull request merged**